java/src/test/java/ai/rapids/cudf/TableTest.java

```diff
@@ -195,6 +195,7 @@ public static void assertPartialColumnsAreEqual(HostColumnVectorCore expected, l
           case UINT32: // fall through
           case TIMESTAMP_DAYS:
           case DURATION_DAYS:
+          case DECIMAL32:
```

Comment:
Same is added in https://github.com/rapidsai/cudf/pull/6734/. If that goes in first, will remove from here.